### DISCUSSION TITLE
3/30(목) 옵셔널 체인징

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#42] [General]
오늘은 옵셔널 체인징에 대해서 공부하시면 될것 같습니다. 앞으로는 코드와 예시를 잘 들어서 예기 해주세요. 개념을 확실히 알고 가는 것이 좋습니다. 문제는 코드를 잘치는 것도 중요하지만 그 부분에 개념을 빠삭하게 알고 가는 것이 중요합니다. 만약 모르거나 개념이 너무 생소하면 불르면 갈꼐요 🌹

**Comment by Eunho0922** [comment]
넹
